lib/cc/analyzer/issue_validator.rb

```diff
@@ -0,0 +1,49 @@
+module CC
+  module Analyzer
+    class IssueValidator
+      CHECKS = [
+        IssueCategoryValidation,
+        IssueCheckNamePresenceValidation,
+        IssueDescriptionPresenceValidation,
+        IssuePathExistenceValidation,
```

Comment:
When we first introduced these, we made the choice that they should 

1. Not be order dependent
2. Be able to run independently of each other
3. All be run all the time

This keeps the classes small (responding to just a `valid?` and `message` method). See that discussion [here](https://github.com/codeclimate/builder/pull/77#discussion_r34052959).

I don't like the idea of doing multiple checks in one validation class - I think it will cause the classes to become bloated and do too many things (additionally, having a single coherent message to be returned from them will be harder)